src/simplewallet/simplewallet.cpp

```diff
@@ -4735,7 +4735,7 @@ bool simple_wallet::register_service_node(const std::vector<std::string> &args_)
   uint64_t locked_blocks = STAKING_REQUIREMENT_LOCK_BLOCKS + STAKING_REQUIREMENT_LOCK_BLOCKS_EXCESS;
 
   std::string err;
-  uint64_t bc_height = get_daemon_blockchain_height(err);
+  uint64_t bc_height = m_wallet->get_daemon_blockchain_target_height(err);
```

Comment:
This needs investigation as I've seen this return 0 before. I handled it like this.
https://github.com/loki-project/loki/blob/07d5e7968fe0892485269d6bd2ab13e9df320853/src/cryptonote_core/cryptonote_core.cpp#L1680

Reply:
Thanks. Fixed.